content/docs/accessibility.md

```diff
@@ -339,14 +335,14 @@ class BlurExample extends React.Component {
     });
   }
 
-  // If a child receives focus, do not close the popover.
+  // Եթե ժառանգը ստանում է ֆոկուս իրադարձություն popover-ը չի փակվում։ 
   onFocusHandler() {
     clearTimeout(this.timeOutId);
   }
 
   render() {
-    // React assists us by bubbling the blur and
-    // focus events to the parent.
+    // React-ը օգնում է մեզ, տարածելով blur և
```

Comment:
"տարածելով"֊ից առաջ բութ ա պետք ստորակետի փոխարեն։

https://manetorunyan.wordpress.com/2017/10/31/%D5%A4%D5%A5%D6%80%D5%A2%D5%A1%D5%B5%D5%A1%D5%AF%D5%A1%D5%B6-%D5%A4%D5%A1%D6%80%D5%B1%D5%BE%D5%A1%D5%AE%D5%AB-%D5%AF%D5%A5%D5%BF%D5%A1%D5%A4%D6%80%D5%B8%D6%82%D5%A9%D5%B5%D5%B8%D6%82%D5%B6/